src/components/Reviews.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Reviews from './Reviews'
+
+export default {
+  title: 'Reviews',
+  component: Reviews,
+}
+
+const Template = (args) => <Reviews {...args} />
+
+export const singleReview = Template.bind({})
+
+singleReview.args = {}
```

Comment:
If you don't want pass args you can leave that out
```suggestion
const Template = () => <Reviews />

export const singleReview = Template.bind({})
```